src/Dfc.CourseDirectory.WebV2/Features/Providers/EditProviderTypeConfirm.cshtml

```diff
@@ -0,0 +1,79 @@
+﻿@using Dfc.CourseDirectory.WebV2.Features.Providers.EditProviderType
+@model ConfirmViewModel
+@{ 
+    ViewContext.SetLayoutData(title: "Update provider type");
+
+    var affectedItemCountErrors = ViewData.ModelState[nameof(ConfirmViewModel.AffectedItemCounts)]?.Errors
+        ?? new Microsoft.AspNetCore.Mvc.ModelBinding.ModelErrorCollection();
+}
+
+<govuk-error-summary>
+    <govuk-error-summary-item asp-for="Confirm" data-testid="confirm-error" />
+    <govuk-error-summary-item asp-for="AffectedItemCounts" data-testid="affected-item-counts-error" />
+</govuk-error-summary>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        <h1 class="govuk-heading-l">Update provider type</h1>
+
+        <p class="govuk-body">By updating the provider type, the following will be permanently deleted and no longer available on Publish to the course directory, Find a course or Tlevels.gov.uk.</p>
+
+        <form asp-action="Post" asp-all-route-data="@ProviderContext.RouteValues">
+
+            @Html.HiddenFor(m => m.ProviderId, new Dictionary<string, object> { { "data-testid", "provider-id" } })
+
+            <input type="hidden" name="@nameof(Model.ProviderType)" value="@((int)Model.ProviderType)" data-testid="provider-type" />
+
+            @for (int i = 0; i < Model.SelectedProviderTLevelDefinitionIds?.Count; i++)
+            {
+                @Html.HiddenFor(m => m.SelectedProviderTLevelDefinitionIds[i], new Dictionary<string, object> { { "data-testid", "selected-provider-tlevel-definition-id" } });
+            }
+
+            @for (int i = 0; i < Model.ConfirmAffectedTLevelIds?.Count; i++)
```

Comment:
Contemplating whether to use a hash of the affected Guids here for performance and scalability... will the number of affected t-levels grow too large for this to be a reasonable postback given the WAF?

Reply:
How about `max(isnull(DeletedOn, CreatedOn))`?

Reply:
That suggestion is essentially replacing a `Guid` with a `DateTime`, correct? There's no guarantee of uniqueness with the `DateTime.`